language-support/qute/language-configuration.json

```diff
@@ -16,6 +16,7 @@
 		{ "open": "\"", "close": "\"" },
 		{ "open": "{!", "close": "!", "notIn": [ "comment", "string" ]}
 	],
+	"autoCloseBefore": ";:.,=}])<> \n\t",
```

Comment:
To be honnest with you, I did that quickly. Don't hesitate to improve it.

Reply:
Ok. @JessicaJHee , the `autoCloseBefore` looks fine to me. It would also apply to the other elements of `autoClosingPairs` so it's worth keeping in mind. However I couldn't find any problems. Could you please also add the case of opening `<` and closing `>` to `autoClosingPairs` ?